javascript/extensible-search-suggestions.js

```diff
@@ -1,35 +0,0 @@
-;(function($) {
```

Comment:
Any reason this is being removed completely? What if someone just wants some basic suggestions instead of the full overlay functionality?